app/views/userProfile.scala.html

```diff
@@ -84,12 +150,15 @@
         }, function(err, t) {
             var difficultRegionIds = @Json.toJson(RegionTable.difficultRegionIds);
             window.progress = Progress(_, $, L, "@user.get.role", difficultRegionIds);
+
+            const achievementTracker = new AchievementTracker();
+            achievementTracker.updateBadgeAchievementGrid(
```

Comment:
I removed the measurement system from the list of parameters, because you can get the measurement system in the Javascript using `i18next.t('common:measurement-system')`